tests/wicked/before_test.pm

```diff
@@ -16,10 +16,12 @@ use warnings;
 use testapi;
 use utils qw(zypper_call systemctl file_content_replace);
 use network_utils qw(iface setup_static_network);
+use serial_terminal 'add_serial_console';
```

Comment:
I'm not 100% sure but `wickedbase` should already contain `add_serial_console` doesn't it?

Reply:
no